### PR TITLE
Added BASIC_AUTH environment variables to acceptance test documentation

### PR DESCRIPTION
**Pull request by Nickersoft**
@clintonb @renzo FYI: @jimabramson @wedaly

### CODE REVIEW

File: acceptance_tests/README.rst [hunk 1]
```diff
@@ -67,10 +67,10 @@ Learning Management System (LMS) Settings
    next to "Trusted Clients".
 
 5. Add an access token for the superuser by clicking the *Add* button
-   next to "Access Tokens". Set the "Client" of the access token to the
```
Comment (clintonb) [inline acceptance_tests/README.rst:70]: What's going on with all of these non-basic auth changes?

Comment (Nickersoft) [inline acceptance_tests/README.rst:70]: @clintonb I think my editor just corrected some tabbing/spacing issues. There was a mix of tabs and spaces and all it did was normalize it.

Comment (Nickersoft) [inline acceptance_tests/README.rst:70]: @clintonb Actually, looking now I see that all it did was remove some trailing spaces. Nothing major.

**Comment by clintonb** [comment]
:+1:

**Pull request merged**